src/client/src/app/components/merch/merch.component.ts

```diff
@@ -14,7 +15,8 @@ import { Merch } from '../../../../../shared/models/merch.model';
 export class MerchComponent implements OnInit {
   merch$: Observable<Merch[]>
   constructor(
-    private store: Store<AppState>
+    private store: Store<AppState>,
+    private router: Router
```

Comment:
```suggestion
    private router: Router,
```
*wanted to see how suggestions work but also i might leave a trailing comma on this